xamarin/Xamarin.Forms
# [iOS] Fixed SwipeItemView rendering issue

### Description of Change ###

Fixed issue having duplicated content on iOS SwipeView.

### Issues Resolved ### 

- fixes #8781

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![70330911-5e713700-1836-11ea-8777-6e1d6778c04f](https://user-images.githubusercontent.com/6755973/70530317-e9ae3d80-1b52-11ea-8242-7a73a098741b.png)

#### After
![issue8781-ios](https://user-images.githubusercontent.com/6755973/70530266-cdaa9c00-1b52-11ea-8c95-bdebca97aba8.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 8781. Open the SwipeView and verify that the SwipeItemView is rendered correctly.

### PR Checklist ###
